build-system/compile/bundles.config.bento.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "name": "jwplayer",
```

Comment:
Will the `bentojs.dev` repository need to know about this?

Reply:
Yes, this is going to have to be part of the build process for the new repo

Reply:
I don't think Kris was talking about the new repo, he was talking about the current documentation repo: https://github.com/ampproject/bento.dev, hosted at http://bentojs.dev

Reply:
TIL there is a bento.dev repository. Yes, it looks like this needs to be updated.

BTW the linked repositories README links to [https://bento.dev/](https://bento.dev/) which seems to be an unrelated project [pr/136](https://github.com/ampproject/bento.dev/pull/136)